example/linux/systemd/Talawa-api.sh

```
@@ -0,0 +1,175 @@
+#!/bin/bash
+# filepath: /path/to/your/talawa-api/example/linux/systemd/Talawa-api.sh
+# Description: Talawa API startup script
+#CODEROOT this is declared as enviroment in service file.When systemd will run this script we will get this value.
+
+#check this path if valid or not
+PROJECT_DIR=$CODEROOT
+LOG_FILE=$LOGFILE
```

Shellcheck (0.10.0)

[info] Possible misspelling: LOGFILE may not be assigned. Did you mean LOG_FILE?

(SC2153)